tests/test_landings.py

```diff
@@ -29,9 +29,13 @@ def db(app):
         _db.drop_all()
 
 
-def test_landing_revision_saves_data_in_db(db, client, phabfactory):
+def test_landing_revision_saves_data_in_db(
+    db, client, phabfactory, transfactory
+):
     phabfactory.user()
     phabfactory.revision()
+    transfactory.land()
```

Comment:
I think this test is missing some data that would make the condition under test clearer.  The user is not important to what's being verified, but I think seeing that a revision with the same ID is being created and being landed is important to seeing what is verified.

Could we add a "revision" parameter to `transfactory` calls so that they aren't using invisible default data to do their work?  And change the language of method so that is implies creating a response rather than executing a process.  Something like this:

```python
phabfactory.user()
revision = phabfactory.revision()
transfactory.add_response(revision, 'pending')
```